schema.json

```diff
@@ -0,0 +1,72 @@
+{
+    "type" : "object",
+     "properties": {
+        "timestamp": {
+            "type": "string",
+	        "pattern": "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+-\\d+$",
+            "optional" : false
+	                },
```

Comment:
minor: indenting seems off here.  vscode shift+ctrl+i seems to clean it up nicely.